src/components/shared/Autocomplete.js

```diff
@@ -127,17 +127,24 @@ export default class Autocomplete extends Component {
     );
 
     const searchProps = {
-      query: this.state.inputValue,
+      query: this.props.inputValue || this.state.inputValue,
       count: searchResults.length,
       placeholder: this.props.placeholder,
       size,
       showErrorEmoji: true,
       summaryMsg,
-      onChange: e =>
-        this.setState({
-          inputValue: e.target.value,
-          selectedIndex: 0
-        }),
+      onChange:
+        (e => {
+          this.props.onChangeHandler(e.target.value);
+          this.setState({
+            selectedIndex: 0
+          });
+        }) ||
+        (e =>
+          this.setState({
+            inputValue: e.target.value,
+            selectedIndex: 0
+          })),
```

Comment:
Hi @rohanprasad  interesting why the need for  the `||`? The second handler will not be used anyway?


Shouldn't this just work ?

```js
onChange: e => {
   this.props.onChangeHandler(e.target.value);    
   this.setState({ 
      inputValue: e.target.value,
      selectedIndex: 0
   });
}
```

Reply:
Hey @bomsy, I couldn't actually find any other place where autocomplete is used but  left that so to prevent breaking other code. What I was thinking was to remove the query string from state to only use the prop. Should I go ahead and do that ?

Reply:
yes that should be fine.